examples/angular/tsconfig.json

```diff
@@ -10,8 +10,9 @@
     "experimentalDecorators": true,
     "moduleResolution": "node",
     "importHelpers": true,
+    "resolveJsonModule": true,
     "target": "es2020",
-    "module": "es2020",
+    "module": "esnext",
```

Comment:
Why are we updating this?

Reply:
Dynamic import requires top level await, which is supported after `module: "es2022"`